src/transformers/modeling_bart.py

```diff
@@ -871,16 +912,24 @@ def forward(
             decoder_padding_mask,
             decoder_causal_mask=causal_mask,
             decoder_cached_states=decoder_cached_states,
+            use_cache=use_cache,
```

Comment:
Reorder arg for consistency across models.

Reply:
thanks!